infra/lib/api-stack/fhir-conversion-checker.ts

```diff
@@ -0,0 +1,70 @@
+import { Duration } from "aws-cdk-lib";
+import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
+import { IVpc } from "aws-cdk-lib/aws-ec2";
+import { Function as Lambda, Runtime } from "aws-cdk-lib/aws-lambda";
+import { Construct } from "constructs";
+import { EnvConfig } from "../env-config";
+import { getConfig } from "../shared/config";
+import { createScheduledLambda } from "../shared/lambda-scheduled";
+
+export type ConversionCheckerProps = {
+  stack: Construct;
+  vpc: IVpc;
+  apiAddress: string;
+  alarmSnsAction?: SnsAction;
+};
+
+function settings(
+  props: ConversionCheckerProps,
+  config: NonNullable<EnvConfig["fhirConversionChecker"]>
+) {
+  return {
+    ...props,
+    name: "ScheduledFHIRConversionChecker",
```

Comment:
nit: would `DocQueryJobChecker` or `DocQueryStatusChecker` be more accurate? This makes it seems like it's only for conversions, but really it's for any doc that is downloaded